### ISSUE TITLE
Audio Switcher Client Issue

### ISSUE THREAD

**Issue by xenolightning** [xenolightning/AudioSwitcher_v1#828]
### Details:
Audio Switcher
Version: 1.6.4.2
Operating System: Microsoft Windows NT 6.2.9200.0 64-bit
Administrator Privileges: True

### User Comments:
I've been having issues with my new wireless headphones, Corsair h2100. They keep reapearing as an active device, randomly switching the audio stream from the speakers to the headphones. Now I just tried to use the volume controls on the headphones, they stoped working and this audio switch error appeared
### Stacktrace:
```
System.NullReferenceException: Object reference not set to an instance of an object.
   at AudioSwitcher.AudioApi.CoreAudio.CoreAudioDevice.get_IsDefaultDevice()
   at FortyOne.AudioSwitcher.AudioSwitcher.RefreshNotifyIconItems()
   at FortyOne.AudioSwitcher.AudioSwitcher.RefreshPlaybackDevices()
```

**Comment by CntZer0** [comment]
I'm having issues without AudioSwitcher open as well, it seems these headphones don't play well with Win8. So it might be that, but regardless, if you could take a look...

**Comment by xenolightning** [comment]
Sounds like it's how the headphones are set up.

It's likely that Windows is switching to them when they connect.

It's outside of Audio Switchers control.

There may be away around it with Audio Switcher 2.0, but that's still a little way from release

**Comment by CntZer0** [comment]
Yeah, seems to be resolved by switching to a 2.0 USB port from 3.0...
